lib/install.js

```diff
@@ -156,7 +150,7 @@ function install (gyp, argv, callback) {
   }
 
   function getContentSha(res, callback) {
-    var shasum = crypto.createHash(checksumAlgo)
+    var shasum = crypto.createHash(release.checksumAlgo)
```

Comment:
Just a suggestion but wouldn't it be easier to generate SHA256SUM.txt files for old releases than special-case here?